backend/src/main/java/ai/verta/modeldb/cron_jobs/ParentTimestampUpdateCron.java

```diff
@@ -24,7 +24,8 @@ public ParentTimestampUpdateCron(int recordUpdateLimit) {
   }
 
   public void initExperimentUpdateQueryString(int recordUpdateLimit) {
-    if (ModelDBHibernateUtil.rDBDialect.equals(ModelDBConstants.POSTGRES_DB_DIALECT)) {
+    if (ModelDBHibernateUtil.config.RdbConfiguration.RdbDialect.equals(
```

Comment:
instead of this if check should we add just 
```
isPostgresDB(){
  return RdbConfiguration.RdbDialect.equals(
        ModelDBConstants.POSTGRES_DB_DIALECT);
}
```
and will used just `isPostgresDB()` at all places

Reply:
Check one of the follow-up PRs ;)

Reply:
Sorry about this comment. i does not show it. https://github.com/VertaAI/modeldb/pull/1814

Reply:
Don't worry. This PR is also useful for you to see what I mean of 1 PR for 1 thing changing. It has ~5 small follow-ups that are not part of the core change.